src/improvements/justfile

```diff
@@ -171,16 +171,97 @@ simulate-all-templates:
         exit 1
     fi
 
-simulate-verify-task TASK="":
+# Simulate a task locally and against Tenderly.
+# TASK: The fullpath to the task to simulate.
+# SAFE_NAME: The name of the safe to simulate the task for (council, foundation). Leave blank for non-nested tasks.
+#
+# Example:
+# cd src/improvements
+# just simulate-tenderly tasks/sep/000-opcm-upgrade-v200 council
+# just simulate-tenderly tasks/sep/003-unichain-superchain-config-fix
+simulate-tenderly TASK="" SAFE_NAME="":
+    #!/usr/bin/env bash
+    set -euo pipefail
+    root_dir=$(git rev-parse --show-toplevel)
+    forge build
+
+    # If the task path doesn't start with a slash, append the current directory to the task path
```

Comment:
This line makes my editor appear misconfigured. Removing the apostrophe helps. 

![CleanShot 2025-03-27 at 16 33 34@2x](https://github.com/user-attachments/assets/46ce6fa8-48a9-4f83-ad7c-fc5e7a5dbc51)
![CleanShot 2025-03-27 at 16 33 44@2x](https://github.com/user-attachments/assets/c65f639f-fb71-45b4-972d-c5d85d806432)

Reply:
Interesting, I use the standard "Shell Script" highlighting from VSCode for justfiles and it works fine. Fixed anyway.